app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list: list[Car]) -> float:
+        income = 0
+        for served_car in cars_list:
+            result = self.wash_single_car(served_car)
+            income += result
+        return income
+
+    def calculate_washing_price(self, washed_car: Car) -> float:
+        comfort_class = washed_car.comfort_class
+        cleaning_operations = self.clean_power - washed_car.clean_mark
+        relevance = self.average_rating / self.distance_from_city_center
+        return round(comfort_class * cleaning_operations * relevance, 1)
+
+    def wash_single_car(self, served_car: Car) -> float:
+        if served_car.clean_mark < self.clean_power:
+            washing_price = self.calculate_washing_price(served_car)
+            served_car.clean_mark = self.clean_power
+            return washing_price
+        else:
+            return 0
```

Comment:
```suggestion

        return 0
```
please, remove redundant else :)